# Introduction

<!-- Chapter 1 -->
## Antimicrobial resistance

Antimicrobial resistance (AMR) is the ability of microorganisms to survive and grow in the presence of antimicrobials [@Balaban2019].  Quinolone resistance is of concern as quinolones are on the list of critically important antimicrobials [@WHO2019]. Resistant bacteria pose a threat to both human and animal health, as fewer treatment options exist. For the animal and food industry this means a loss of productivity and livelihood [@WHO2015]. Antimicrobial usage has been linked to an increased occurrence of resistant bacteria [@WHO2018b]. In Norway, antimicrobial usage is low among livestock, and quinolones are hardly used [@NORM/NORM-VET2018; @ESVAC2018] (Figure \@ref(fig:quinUsage)). This is reflected in the good animal welfare situation [@Alvseike2018], as good health reduces the need for antimicrobial treatment. In this chapter, the broiler and pig production in Norway is presented. Antimicrobial usage within these populations is briefly described. The chapter then gives an overview of the occurrence of quinolone resistance in Norway and Europe. Finally, quinolone resistance mechanisms are presented.

```{r, fig.cap="\\label{fig:quinUsage}Overall sales, in kg active substance, of quinolones and macrolides for therapeutic use in terrestrial livestock animals in Norway in 1993 - 2018, reprinted with permission from NORM/NORM-VET 2018 \\citep{NORM/NORM-VET2018}.", echo=FALSE, message=FALSE, fig.pos="H", fig.align="center", out.width="100%", fig=TRUE}
knitr::include_graphics("images/pdfs/quin_usage.pdf")
```

### Norwegian animal husbandry
#### Broiler production
The Norwegian broiler production is part of a pyramidal structure with pure-bred animals at the top, breeding animals in the middle, and broilers at the bottom [@Mo2016a]. The pure-bred animals originate from Scotland or Germany. Eggs that are laid by these pure-bred animals are imported to Sweden and become grandparent animals. The grandparent animals lay eggs that are imported to Norway. These eggs are hatched in Norwegian hatcheries and become parent animals. The parent animals are reared for around 18 weeks, and are subsequently used for breeding of broiler chickens until around 60 weeks old. The broiler chickens are hatched in a few hatcheries, and distributed to broiler farms all over the country. Most of the broiler chickens are reared for 28 - 32 days, depending on weight, then slaughtered [@Nortura2015]. The broiler farms have high biosecurity, i.e. measures are taken to prevent exposure to harmful agents.

Minimal levels of antimicrobials are used in the Norwegian broiler production [@NORM/NORM-VET2018]. During the last six years, only 0.02 - 0.18% of all broiler flocks have been treated with penicillins [@NORM/NORM-VET2018]. Quinolones are not used in the Norwegian broiler production (personal communication: HÃ¸y T., The Norwegian Medicines Agency, October 2017). 

#### Swine production
Swine production in Norway has a pyramidal structure with unidirectional flow of animals, similar to the broiler production [@Kristoffersen2016]. Import of live pigs from other countries to the commercial Norwegian pig population is negligible [@Animalia2018]. The genetic nucleus herds, where the pure-bred grandparent animals are produced, are at the top of the pyramid. Hybrid parent animals are produced in multiplier herds, either by recruiting pure-bred animals from nucleus herds or having a self-recruitment strategy. Gilts from the multiplier herds are distributed to piglet producing herds (either farrow to finish or farrow to grower herds). The piglet producing herds have a mean weaning age of 33 days, followed by a growing stage until the grower pig is approx. 30 kg and 10-11 weeks of age. The grower pig is then transferred to a specialized fattening pig farm or fattening unit at the same farm.

The data reported to the Veterinary Prescription Register on prescription and use of antibiotics for terrestrial food-producing animals are not complete. These represent approximately two thirds of the sales of injectables, oral powder, and oral solution [@NORM/NORM-VET2018]. However, the percentage use of each antimicrobial type, as kilograms of active substance, is known. Of all prescribed antimicrobials to pigs in 2018, only 0.1% were quinolones, compared to 86.8% for penicillins [@NORM/NORM-VET2018]. Quinolones are therefore used at a very low level in Norwegian pigs.

### Monitoring AMR and QREC epidemiology
#### The situation in Europe

Harmonized monitoring of AMR in commensal _E. coli_, i.e. _E. coli_ that is naturally present in the gut, from food and food-producing animals has since 2014 been obligatory for all EU/EEA member states. This was implemented by the European Commission (SANCO/11591/2012). The commission describes the sampling frame and analysis of isolates. Commensal _E. coli_ is usually isolated from caecal samples. Antimicrobial susceptibility testing (AST) is carried out using broth microdilution. With this method, the minimum inhibitory concentration (MIC) value is determined for multiple antimicrobials. A MIC value is defined as the lowest concentration of the antimicrobial needed to inhibit growth of a micro-organism [@Andrews2001]. Based on the MIC value, an isolate may be categorized as either susceptible or resistant. This is based on breakpoints defined by the European Committee on Antimicrobial Susceptibility Testing (EUCAST). In a clinical setting, the clinical breakpoint is used to determine if the treatment will be successful [@Aarestrup2007a]. However, for monitoring purposes, the epidemiological breakpoint (ECOFF) is frequently used. The ECOFF is the highest MIC value for the susceptible (wild type) population [@Aarestrup2007a;@EFSA2008], see Figure \@ref(fig:ecoff). It is important to distinguish between these two breakpoints. Isolates categorized as resistant with the ECOFF value may still respond to treatment in a clinical setting. Using the ECOFF enables detection of resistant isolates before the potential development of clinical resistance. The ECOFF therefore works as a precautionary measure.
\
```{r, fig.cap="\\label{fig:ecoff}Distribution of MIC-values for ciprofloxacin in \\textit{E. coli}. Data: www.eucast.org.", echo=FALSE, message=FALSE, fig.pos="H", fig.align="center", out.width="110%", out.height="110%"}
knitr::include_graphics("images/pdfs/micdist.pdf")
```

\newpage

Occurrences are often reported as a percentage, but can sometimes be referred to with terms such as "low" and "high". Guidelines from EFSA and ECDC specify the percent occurrence that correspond to each of these terms [@EFSA2016]. The corresponding terms and percentages are "rare" (< 0.1%), "very low" (0.1 - 1%), "low" (> 1 - 10%), "moderate" (> 10 - 20%), "high" (> 20 - 50%), "very high" (> 50 - 70%), and "extremely high" (> 70%).

The occurrence of quinolone resistance among commensal _E. coli_ in Europe seems to follow a north - south gradient [@EFSA2017; @EFSA2016]. The northern countries seem to have a low occurrence while the southern countries a higher occurrence (Figure \@ref(fig:ecdcData)). The overall occurrence of QREC from 2006 to 2017 from production animals including pigs, cattle less than one year old, and various poultry animals was 26.1%. More specifically, the EU countries had a mean QREC occurrence of 49.67% in poultry from 2006 to 2016 [@EFSA200407; @EFSA2008; @EFSA2009; @EFSA2010; @EFSA2011; @EFSA2012; @EFSA2013; @EFSA2014; @EFSA2016]. This level corresponds to a high occurrence. A mean occurrence of 8.73% was observed in pigs within the same time frame [@EFSA200407; @EFSA2008; @EFSA2009; @EFSA2010; @EFSA2011; @EFSA2013; @EFSA2015], corresponding to a low occurrence. To the authors' knowledge, no systematic monitoring of wild animals have been done at a European level.

```{r, fig.cap="\\label{fig:ecdcData}Mean occurrence of quinolone resistance in commensal \\textit{E. coli} from 2011 - 2017. Mean values from broilers was based on data from 2011, 2012, 2013, 2014, and 2016, for pigs from 2011, 2012, 2013, 2015 and 2017, for calves from 2015 and 2017, and for turkeys from 2014 and 2016. Grey areas represent missing data. Data taken from European Union summary reports on antimicrobial resistance in zoonotic and indicator bacteria from humans, animals and food 2011 - 2017 \\citep{EFSA2013,EFSA2011,EFSA2012,EFSA2013,EFSA2014,EFSA2015,EFSA2016,EFSA2017}.", echo=FALSE, message=FALSE, fig.pos="H", fig.align="center", out.width="90%", out.height="90%"}
knitr::include_graphics("images/pdfs/ecdcdata2.pdf")
```

#### The situation in Norway {#norwmonitor}
As part of the Norwegian government's strategy plan against antimicrobial resistance, the Norwegian monitoring programme for antimicrobial resistance in feed, food and animals (NORM-VET) was implemented in 2000. The programme has since 2014 followed the recommendations for surveillance as given by the EU (2013/652/EU) in parallel with national surveillance needs. A variety of animal species have been sampled in NORM-VET, but all animals are not sampled each year due to the sampling regime. The sample material is commonly faecal or caecal matter and meat, but boot swab samples are also included some years. In broiler and turkey flocks, 10 caecal samples are collected from each flock and pooled before culturing. For pigs and cattle, one individual is sampled per herd. Wild animals are sampled individually. Standardized culturing and isolation methods are used to identify _E. coli_ from the samples collected. These isolates are used as an indicator for the presence of resistance in the given animal population on a national level. Quinolone resistance in commensal _E. coli_ has been monitored in NORM-VET since 2000. Here, samples are plated onto lactose-bromothymol blue agar (pre-2013) or MacConkey agar (2013 until today). A single colony of presumptive _E. coli_ is randomly selected for susceptibility testing, hereafter referred to as the traditional method.

Compared to the overall occurrence of quinolone resistance among indicator _E. coli_ in the EU, the overall occurrence in Norway is low. For instance, the mean occurrence of QREC between 2006 and 2018 was 5.22% in broilers, and 0.3% for pigs [@NORM/NORM-VET2006; @NORM/NORM-VET2007; @NORM/NORM-VET2009; @NORM/NORM-VET2011; @NORM/NORM-VET2012; @NORM/NORM-VET2014; @NORM/NORM-VET2015; @NORM/NORM-VET2016; @NORM/NORM-VET2017; @NORM/NORM-VET2018], corresponding to a low and very low occurrence, respectively. However, a peak in QREC occurrence was detected in 2009 at 8% [@NORM/NORM-VET2009], with no apparent explanation at that time. This triggered the introduction of a selective method in 2014 to further investigate this unexplained occurrence. With the selective method, the samples are plated onto MacConkey agar with 0.06 mg/L ciprofloxacin. Analysis with this method revealed that QREC were present in 89.5% of included broiler caecal samples in 2014. Corresponding analyses on samples from other animal species, such as pigs, red foxes, wild birds, breeder flocks and horses were performed in 2015, 2016 and 2017. Here, the occurrences ranged from 2.4% in horses, 5.6% in wild birds, 14.8% in red foxes, 54.3% in pigs to 100% in broiler breeder flocks [@NORM/NORM-VET2015; @NORM/NORM-VET2016; @NORM/NORM-VET2017]. In comparison, the occurrence of quinolone resistance among commensal _E. coli_ identified with the traditional method in the same samples was 0% in horses and breeder flocks, 1.2% in red foxes, 2.3% in wild birds, and 0.7% in pigs. This difference in occurrence between the two methods indicates that the QREC are generally present at low levels.


### Quinolones
Quinolones were introduced to the market in the late 1960s [@Emmerson2003]. The first quinolone, nalidixic acid, was discovered as a by-product of chloroquine synthesis [@Lesher1962], and was clinically in use in 1967. A few decades later, the synthesis of novel quinolones increased the effectiveness of nalidixic acid towards Gram positive bacteria, and reduced its toxicity by altering its chemical structure. This structural change involved, among others, the addition of a fluoro-group to the molecule, which also increased its potency towards Enterobacteriaceae [@Emmerson2003]. This also lead to a change in nomenclature for these compounds, as the molecules that harboured this structural change were then called fluoroquinolones.

The main target for quinolones and fluoroquinolones (hereafter called quinolones) in _E. coli_, is the type II topoisomerases DNA gyrase (also known as topoisomerase II) and topoisomerase IV [@Drlica1997]. The type II topoisomerases regulate DNA topology by cutting DNA strands, passing another DNA strand through the break, and then sealing the break [@Hooper1998]. Both DNA gyrase and Topoisomerase IV are comprised of two subunits, A and B. These are encoded by the genes _gyrA_ and _gyrB_ for DNA gyrase, and _parC_ and _parE_ for topoisomerase IV. Quinolones bind to these enzymes through a water-metal ion bridge (Figure \@ref(fig:watermetalion)), connecting the quinolone molecule to two specific amino acids; serine (S) 83 and aspartic acid (D) 87 in DNA gyrase, and S 80 and glutamic acid (E) 84 in topoisomerase IV [@Aldred2014]. This binding inhibits the ligase activity of the enzyme, and fragments the bacterial chromosome, leading to cell death.

```{r, fig.cap="\\label{fig:watermetalion}Overview of the water-metal ion bridge that mediates the quinolone-topoisomerase IV interaction, reprinted from Aldred et al. 2013 \\citep{Aldred2013} with permission from Nucleic Acids Research.", echo=FALSE, message=FALSE, fig.pos="H", fig.align="center", out.width="50%", out.height="50%"}
knitr::include_graphics("images/pdfs/watermetalion.pdf")
```


### Quinolone resistance mechanisms in _E. coli_
Resistance mechanisms may be disseminated either by vertical inheritance by cell division or through horizontal transfer of genetic material from one cell to another. Multiple mechanisms that confer resistance towards quinolones have been discovered in _E. coli_, and they are generally divided into four categories: chromosomal mutations, reduced membrane permeability, efflux pump overexpression, and plasmid mediated quinolone resistance (PMQR) [@Correia2017], see Figure \@ref(fig:mechanisms). Except for PMQR, these resistance mechanisms are mediated by mutations in the chromosome that either lead to structural changes of the encoded proteins or changes in expression through various pathways. The four categories are therefore described as two different categories below: chromosomal mutations and PMQR.

```{r, fig.cap="\\label{fig:mechanisms}Quinolone resistance mechanisms in \\textit{E. coli}. Reprinted from Correia et al. 2017 \\citep{Correia2017} with permission from the Microbiology Society.", echo=FALSE, message=FALSE, fig.pos="H", fig.align="center", out.width="100%",out.height="100%"}
knitr::include_graphics("images/pdfs/mechanisms.pdf")
```


#### Chromosomal mutations
Perhaps the most well-studied resistance mechanism towards quinolones is through mutations in the genes that code for the target molecules, _gyrA_, _gyrB_, _parC_, and _parE_. Missense mutations in these genes that lead to a substitution of critical amino acids needed for the binding of quinolones to the target molecules may lead to an increased tolerance towards these compounds. In _E. coli_, substitutions in a stretch of amino acids known as the quinolone resistance determining region (QRDR) in these four genes is highly correlated with quinolone resistance. In GyrA, this region is between amino acid (AA) 67 to 106 [@Yoshida1991], in GyrB between AA 333 and 481, ParC between AA 51 and 170, and ParE between AA 366 and 523. Specific substitutions within these areas are often identified in QREC, e.g. the S 83 and D 87 positions in GyrA [@Barnard2001], or the S 80 position in ParC.

Porin channels in Gram negative bacterial membranes are essential for the entry of quinolones into the cell [@Correia2017]. Specific chromosomal regulons are involved in the expression of these membrane proteins, exemplified by the _mar_, _soxRS_ and _rob_ regulons [@Correia2017]. Mutations in these genes may decrease the expression of the Omp-type porins [@Koutsolioutsou2005;@White1997].

In _E. coli_, the AcrAB-TolC efflux pump has been identified as a major facilitator of quinolone efflux [@Sato2013;@Weston2017]. Mutations in specific regulatory genes such as _marR_ may activate _acrAB_ and _tolC_, thus increasing their expression [@Alekshun1999a]. This will in turn reduce the drug concentration in the cell. Certain mutations in RNA polymerase subunit B (_rpoB_) have been shown to increase the expression of MdtK, which is a multidrug efflux transporter that can reduce the intracellular quinolone concentration [@Pietsch2017].

#### Plasmid mediated quinolone resistance
Some proteins that confer resistance to quinolones are transcribed from plasmids, such as the quinolone resistance proteins (Qnr), the efflux pumps OqxAB and QepA, and the acetyltransferase aac(6')-_Ib-cr_.

Qnr proteins are pentapeptide proteins that are capable of protecting DNA gyrase and/or topoisomerase IV from quinolone action. Qnr proteins bind to topoisomerases and prevent the quinolone from stabilizing the lethal gyrase-DNA-quinolone cleavage complex [@Tran2005;@Tran2005a;@Robicsek2006], effectively reducing the possible targets for the quinolones. Multiple subtypes of Qnr proteins have been identified, namely QnrA, QnrB, QnrC, QnrD, QnrS, and QnrVC [@Strahilevitz2009]. Other PMQR genes have been identified, such as _mcbG_, which code for pentapeptide proteins thought to have similar effect as Qnr proteins [@Strahilevitz2009;@Montero2001;@Hegde2005].

The efflux pump OqxAB has been shown to increase resistance towards chloramphenicol and quinolones, and is dependent on the host TolC outer membrane protein in _E. coli_ [@Hansen2004;@Hansen2007]. Similarly, QepA expression has been found to increase resistance towards ciprofloxacin, norfloxacin and enrofloxacin due to efflux [@Yamane2007].

The acetyltransferase aac(6')-_Ib-cr_ infers resistance towards quinolones by altering the quinolone molecule itself, which reduces the activity of the quinolone [@Robicsek2006a].

\newpage
<!-- Chapter 2 -->
## _Escherichia coli_
_E. coli_ is the most studied micro-organism on Earth [@Vila2016], and is often used as a model organism in various microbiology studies and as an indicator for the presence of resistance in the intestine. _E. coli_ is a Gram negative, rod-shaped, facultative anaerobic bacterium often found in the gastrointestinal tract of warm-blooded animals and humans. There, it may be present as a commensal or as a pathogen.

### Genetics of _E. coli_
_E. coli_ is a highly diverse species with over 7000 defined sequence types [@Wirth2006]. Several phylogenetic groups have been identified, representing major lineages within the _E. coli_ species (Figure \@ref(fig:popstructure)). A huge difference in gene content have been detected among _E. coli_ isolates. The pan-genome, i.e. the total amount of genes identified among isolates in a population, can for example range from 15.000 - 40.000 genes [@Her2018; @Yang2019], depending on the number of included isolates [@Rasko2008]. The genes that are present among at least 99% of the included isolates, i.e. the core genes, seem to converge on approximately 2000 genes [@Rasko2008; @Touchon2009]. With this apparent genome plasticity, recombination, i.e. the incorporation of horizontally transferred genes or the rearrangement of chromosomal segments, has definitely been important in the evolution of _E. coli_ [@Chaudhuri2012]. Early studies on the population structure of _E. coli_ presented evidence for a clonal evolution based on electrophoretic movement of enzymes [@Milkman1973; @Selander1980], later supported in studies using sequencing methods [@Touchon2009]. Most studies investigating the population structure and evolution of _E. coli_ seem to conclude that the species seem to evolve in a clonal manner, regardless of it's relatively high rate of recombination [@Chaudhuri2012]. Recombination is further discussed in Section \@ref(phylo).

```{r, fig.cap="\\label{fig:popstructure}Phylogenetic groups of \\textit{E. coli}, based on maximum likelihood analysis of a concatenated alignment of 2173 genes, reprinted from Chaudhuri et al. 2012 \\citep{Chaudhuri2012} with permission from Infection, Genetics and Evolution.", echo=FALSE, message=FALSE, fig.pos="H", fig.align="center",out.height="110%",out.width="110%"}
knitr::include_graphics("images/pdfs/pop_structure_chaudhuri.pdf")
```

\newpage

<!-- Chapter 3 -->
## Sequencing and bioinformatic analyses
In this project, we wanted to investigate the evolutionary relationship between isolates in addition to identifying resistance genes and mutations. For these purposes, we used high-throughput sequencing (HTS) to sequence the genomes of the included isolates and subsequently utilised the bioinformatic methods described below. Bioinformatics can be defined as the use of informatics techniques, mathematics, statistics and computer science to understand biological data on a large scale [@Luscombe2001]. In this chapter, the concept of high throughput sequencing (HTS) and gene identification is presented, followed by a description of the analysis pathway used in this thesis to go from reads to assembly. Then, various subsequent analyses are described, such as pan-genome analysis and phylogenetics.

### High throughput sequencing
The process of Illumina HTS builds upon that of Sanger sequencing. First, genomic DNA is extracted and purified. The gDNA is then fragmented and amplified through a process called library preparation. In this process, the gDNA is fragmented and adapters are placed on each end of each DNA fragment. Then, the fragments are amplified, usually by PCR. Then, the prepared fragments are placed on a flow-cell and the sequencing begins. The machine detects and registers each nucleotide through their unique fluorescent signal. The fragments are polymerized to a given length, usually between 100 - 300 base pairs [@Illumina2019]. Following sequencing, the reads can be used in a multitude of analyses, briefly summarised in Figure \@ref(fig:bioinfo).

```{r, fig.cap="\\label{fig:bioinfo}Main analysis pathway for culture-based genomic data, modified from Boolchandani et al. 2019 \\citep{Boolchandani2019} with permission from Springer Nature.", echo=FALSE, message=FALSE, fig.pos="H", fig.align='center', out.width="110%"}
knitr::include_graphics("images/pdfs/bioinfo.pdf")
```


### Computational identification of genes
Computational identification of genes is mainly done in two ways, either from assemblies or from raw reads [@Boolchandani2019]. In both cases, a database containing the genes of interest is needed. Only the genes that are represented in the database can be detected. In assembly based methods, the references in the database are compared to the assemblies. A specific threshold of similarity is used to define the two sequences as the same gene. A popular method for gene identification with assemblies is the Basic Local Alignment Search Tool (BLAST) [@Altschul1990]. In read based methods, the raw reads are mapped to the references, either directly as whole reads or broken down as k-mers of length _k_. Here, the mapped reads or k-mers are locally assembled and compared to the reference. Identifying genes and mutations by using whole assemblies is computationally demanding while read-based methods using k-mers are faster [@Boolchandani2019].


### From reads to assembly
Reconstructing genomes is an important part of comparative genomics. Having a plausible reconstruction of each bacterial genome opens up the possibility of comparing genomic elements between isolates, and therefore makes it possible to deduce the relationship between them. This section briefly describes the analysis process from quality control of reads to the final assemblies.

Quality control of nucleotides in sequencing reads is important to make sure that the underlying data is of adequate quality for assembly. Quality parameters in fastq files, which is the common format for Illumina data, can be checked with software like fastQC [@BabrahamBioinformatics2018]. The fastq files include quality information for each nucleotide in each read. Reads are trimmed to ensure that only high-quality nucleotides are included, and that residual adapter sequences from the sequencing reaction are removed. After trimming, the reads are ready to be assembled.

Genome assembly is the process where sequence reads are put together into longer, contiguous sequences called contigs, based on overlapping sequences in the reads. Most assemblers in use for Illumina data today are based on de Bruijn graph algorithms [@Simpson2015]. De Bruijn graph algorithms are based on separating the reads into _k_-mers of a specific length, then creating a graph by identifying overlaps. Then, the algorithm "walks" through the graph, and identifies the optimal path where each edge in the graph is visited only once, also known as an Eulerian path [@Pevzner2001], see Figure \@ref(fig:debruijn). This optimal path then becomes the assembly sequence. It is important to note that genome assemblies are only plausible reconstructions of the original genomes in the cells from which the DNA was extracted, and is therefore only a computed approximation of the "real" genome.

```{r, fig.cap="\\label{fig:debruijn}The two de Bruijn graph based assembly methods for genome assembly: Hamiltonian cycles and Eulerian cycles. Reprinted from Compeau et al. 2011 \\citep{Compeau2011} with permission from Nature Biotechnology.", echo=FALSE, message=FALSE, fig.pos="H", fig.align="center", out.width="110%",out.height="110%"}
knitr::include_graphics("images/pdfs/debruijn.pdf")
```


### Post-assembly analyses
Genome annotation is the process where relevant features in the assembled genome is identified and labelled [@Richardson2013]. A commonly used procedure of annotating the genome is to first predict genes from the assemblies using software that identifies coding regions, such as open reading frames (ORFs). Then, the predicted genes are compared to a database of known genes and the most significant match is identified by alignment methods. The annotation of this match is then transferred to the predicted gene [@Richardson2013; @Seemann2014]. Annotating the genome enables the identification of core genes [@Page2015]. This is done with a pan-genome analysis, which is a method of grouping genes based on their presence among the included genomes. For example, core genes are present in 99% of all included genomes. Identifying core genes is useful, as it highlights which genes are highly conserved among the included genomes.

An alignment of core genes can subsequently be used to analyse the genetic relationship between the isolates. In an alignment, the included sequences are assumed to be homologous. The alignment contains blocks of highly similar sequences, and gaps where sequences are either missing or highly divergent. To be able to determine patterns of change between sequences, homologous sequences need to be aligned to be able to compare differences between them correctly [@Higgins2009]. Therefore, aligning sequences is the first step in a phylogenetic analysis, i.e. identifying the evolutionary history of the isolates [@Biology-Online2019].

### Phylogenetic analysis {#phylo}
A frequent problem encountered by bacteriologists is to identify the evolutionary relationship between isolates. Very closely related isolates are sometimes called clones. The term clone does not have a clear definition, but can be loosely defined as isolates that with high probability originated from a recent common ancestor. However, the definition depends on the method used and previous knowledge about the circumstances in which the isolates were involved.

Phylogenetics is the field of study of evolutionary relationships among groups of organisms [@Biology-Online2019]. The most common way of representing phylogenetic relationships between organisms is a phylogenetic tree, exemplified in Figure \@ref(fig:popstructure). Every node in the tree represents the common ancestral state, and the leaves in the tree are the descendants. Trees like these may be calculated from many different types of data, and morphology has historically been frequently used to assess the relationship between the included organisms. However, today, molecular data is more commonly used to generate these trees. For example, an alignment of the core genome is regularly used to identify differences, i.e. single nucleotide polymorphisms (SNPs) between the included organisms. One method of creating these trees from the core genome alignment is the "character-state" method, where each position in the alignment (the "character") is independently analysed in regard to which nucleotide is present in that location (the "state") [@Vandamme2009]. However, to properly make assumptions about the evolutionary relationship between isolates, a model of evolution has to be used in the analysis [@Posada2009]. The choice of model indicate which assumptions are being made for the isolates included in the analysis, such as the rate of evolution, i.e. the mutation rate, for all included isolates. Many models exist, and finding the optimal model can be a difficult task. Therefore, software used for tree reconstruction sometimes has algorithms that find the optimal model to apply to your data to make this task easier. However, this selected model may not be optimal for a subset of the included isolates, since some lineages may evolve at a different rate [@Posada2009]. 

Phylogenetic analysis is based on genealogy, and therefore assumes vertical inheritance. Horizontal gene transfers introduce genes that may have a different evolutionary history than the vertically inherited genome of the isolates. Recombination and horizontal gene transfer does not seem to affect tree topology for _E. coli_, but it does have an effect on branch lengths [@Denamur2010a], i.e. the representation of evolutionary time in the tree. Identifying and removing recombinant areas and horizontally transferred genes is therefore important, not only to prevent overestimated branch lengths, but also to ensure that only vertically inherited parts of the genome is included in the analysis.

Millions of different tree structures can describe the underlying data, and generating these trees is highly computationally demanding. Maximum likelihood (ML) methods are often implemented to handle these problems. Software that use ML methods apply a heuristic approach to identify a reasonably good tree by searching for the tree that maximizes the probability of observing the data, given the selected model of evolution [@Vandamme2009]. Thus, multiple trees are generated, but only the one with the "best fit" to the data is selected.
