# Discussion

## Methodological considerations

### Selection bias

In the epidemiological study (paper I), the entire population of _E. coli_ and QREC obtained through the NORM-VET programme from 2006 to 2016 was included. The sampling performed in NORM-VET from livestock animals at slaughterhouses is based on a proportional sampling according to the slaughter volume at each slaughterhouse. The sampling is done in a random week of the year, with only one sample per flock or herd. These samples can be considered as representative for the livestock populations in Norway. For the wild animals, the sampling is dependent on hunters or other people sending in carcasses to be included in the programme. Therefore, these samples may not be randomly distributed throughout Norway, even with efforts to request samples from the whole country. Regardless, the isolate collection from the wild animals is likely as representative for the actual _E. coli_ population in the respective animal species as possible, and was regarded as such in the epidemiological study.

In paper II, a random selection of isolates was performed after grouping the isolates on phenotypic resistance patterns. This was done to ensure a high phenotypic diversity among the selected isolates. This selection may therefore not represent the actual QREC populations in each included animal species. 

In paper III, the isolate selection included both QREC and quinolone susceptible (wild type) _E. coli_. Here, the goal was to further investigate dissemination, as well as to identify possible development of quinolone resistance in wild type isolates. Isolates were included if the production site was sampled at least three times, and at least one QREC isolate and one wild type isolate were detected in samples from the respective production site. The results may therefore be biased because of the uneven representation of each production site. In addition, not all production sites were represented.

### Genomic analyses methods
The various software used in the genomic analysis were selected because they are internationally recognized programs that are regularly used for such analyses. The programs have a large user base. Thus, their behaviour and result characteristics are well known.

#### Contaminants
MALDI-TOF was used to confirm the species as _E. coli_. The samples could nonetheless have been contaminated during DNA extraction. Colonies may grow on top of each other or have similar morphologies, making them difficult to distinguish. Implementing _in silico_ screening for contaminants is therefore useful to prevent the inclusion of such contaminated data into further analysis. This was exemplified by the exclusion of four isolates in the current study. These samples were found to be contaminated with _Citrobacter_ or _Enterobacter_ using Mash. It is likely that contaminants were present in other included isolates as well. However, in those instances the potential contaminants were likely to be of low presence among the _E. coli_ reads. Thus, the contaminant reads, if present, were likely assembled into small contigs. After the assembly, contigs smaller than 500 bp were filtered out of the data. This step probably removed most of these potential contaminant contigs.

#### Identification of genes and mutations
In this project, ARIBA [@Hunt2017] was used to identify resistance genes and mutations, and for sequence typing. For this work, ARIBA has multiple advantages, such as being able to detect both resistance genes and mutations. ARIBA is a read-based program. Using a read-based approach for the detection of AMR genes has been found to be superior to using assembled sequences [@Clausen2016]. The process of genome assembly is complicated, and if one gene is for example split over multiple contigs they may be missed [@Hunt2017;@Clausen2016]. Another advantage of using ARIBA is the possibility of downloading newly updated databases of our choosing. Here, the ResFinder [@Zankari2012] database was used for identification of acquired genes. The MegaRes [@Lakin2017] database was used for the identification of mutations in chromosomal genes. Resfinder is a popular database for acquired resistance genes, which is manually curated and updated regularly. MEGARes was originally created for metagenomic characterization of AMR genes and mutations [@Lakin2017]. This database contains references from ResFinder, ARG-ANNOT, the Comprehensive Antibiotic Resistance Database (CARD), and the National Center for Biotechnology Information (NCBI) Lahey clinic $\beta$-lactamase archive. MEGARes is also manually curated, and is specifically annotated for high-throughput data processing [@Lakin2017]. MEGARes was chosen as a database because it already encompasses other popular resistance databases. It is also easily downloaded through ARIBA. The results generated by ARIBA using the MEGARes database were easier to interpret than the data generated by using the CARD database alone. This is likely because the annotation is specifically designed for high throughput data processing. 

Databases confer limitations on the resulting data generated by using them. As mentioned earlier, it is only possible to detect the genes that are present in the database, and potential novel genes are therefore missed. In this study, the flag accompanying each predicted gene or mutation were investigated. Genes or mutations that did not have sufficient quality, as presented in section \@ref(aribaflag), were removed. This could, for example, be because the gene was not adequately assembled, or that multiple contigs in the assembly matched the reference. Thus, some genes or mutations may be false negatives. In this project, only novel mutations were confirmed by identifying the same mutation using assembly methods. This was not done, however, for already known mutations, such as the mutation leading to the S83L amino acid substitution in GyrA. One way of checking for potential false negatives from the ARIBA results would be to confirm with assembly based methods. This could have been done with tools such as PointFinder [@CGE2019].

### Phylogeny
Phylogenetic methods were used to determine the evolutionary relationship between the isolates. Core genome SNP phylogeny is regarded as one of the methods with the highest resolution for inferring evolutionary relationships, compared to the clustering method core genome MLST (cgMLST). cgMLST is similar to MLST but utilize a scheme that includes over 2000 genes. Core genome SNP phylogeny and cgMLST have been regarded as complementary in an outbreak setting [@Jagadeesan2019; @Pearce2018]. This is likely not the case when the isolates are not assumed to be closely related, as reflected in the current project. Thus, a phylogenetic approach is appropriate to get the resolution necessary to be able to deduce the relationship between these isolates. 

Genomes were assembled with SPAdes prior to phylogenetic analysis. When dividing each read into _k_-mers, the difficulty of putting them back together is increased by the _k_-mers that contain sequencing errors. These _k_-mers will create diverging paths in the de Bruijn graph, further complicating the assembly. A perfect eulerian path is not possible, as sequencing errors and repeats obscure the graph [@Pevzner2001]. These errors may introduce or mask SNPs or indels in the assembly, but were likely removed during the assembly process. In this study, Pilon was used to correct potential errors in the assembly process by mapping the reads back to the assembly. Thus, sequencing errors were likely corrected. 

Removing recombinant sites is an important step when analysing vertical inheritance, because these sites may have a different evolutionary history. Here, Gubbins was used in this regard. Gubbins have been found to be much faster than similar software [@Croucher2015], such as ClonalFrameML [@Didelot2015]. Recombinant sites are detected by scanning for loci that contain a higher density of base substitutions that the vertically inherited genome. This is indicative of horizontal transfer [@Croucher2015]. Here, assembly- or sequencing errors may influence the results, as Gubbins cannot distinguish between elevated densities generated by horizontal transfer or other causes [@Croucher2015]. Thus, some areas in the genome assemblies may have been erroneously categorized as recombinant and removed. However, the alignments that were used to create the phylogenetic trees were regarded as large enough that this probably did not change the phylogenetic structure in significant ways.

IQTree was selected as the ML algorithm to find the optimal phylogenetic tree.  Comparative analyses have shown that the search strategy implemented in IQTree achieves higher likelihoods than similar algorithms [@Nguyen2015]. IQTree also seem to find optimal trees faster. However, IQTree was not always the best algorithm when tested against RAxML and PhyML. Therefore, the authors recommended using all three programs [@Nguyen2015]. In this study, only IQTree was used, which means that potential trees with higher likelihoods may have been missed.

An evolutionary model was selected by using ModelFinder plus [@Kalyaanamoorthy2017] implemented in IQTree. ModelFinder finds the optimal evolutionary model for the data, based on, for example, Akaike's Information Critera (AIC) and Bayesian Information Criteria (BIC) [@Kalyaanamoorthy2017]. Both AIC and BIC are values that describe how well the model fits the data [@Yang2006]. Low values are preferred for both measures [@Yang2006]. By using these criteria, ModelFinder plus identifies the model that best fit the data. However, using more than one model may be necessary when working with many isolates from potentially different lineages. Some lineages may evolve differently than others. The selected models in this study may therefore not be optimal in describing the evolutionary rate of all the isolates in one tree. This is also partly why phylogenetic analysis was performed separately on more closely related isolates, as exemplified by the ST117 isolates in paper II and ST355 isolates in paper III. Other more appropriate evolutionary models were selected for these trees.

### Statistical considerations

A non-parametric iteration test was used to assess the aggregation of samples in phylogenetic trees based on SNP-distances. A non-parametric test was selected because the underlying distribution of SNP-distances did not follow a theoretical distribution, such as a normal distribution. Additionally, the data could not be regarded as independent observations. Therefore, a test that does not rely on distribution assumptions or independent observations was selected. 

Non-metric multidimensional scaling (NMDS) was used to cluster isolates based on presence / absence of resistance genes and mutations (paper II). NMDS is an ordination technique that differs from other ordination methods, such as principal component analysis (PCA), as NMDS does not assume a relationship between the samples [@Holland2008]. Therefore, any distance measure is allowed in NMDS, which makes this method well suited for a wide variety of data. NMDS is an iterative method, and running the same analysis several times on the same data may yield a slightly different result each time. Therefore, the solution found in paper II may thus represent an adequate solution rather than the "best" one. A stress value is calculated for each iteration and represents how good the data is summarized by the ordination. In the present study, this stress value was low (< 0.05), which is regarded as a good representation [@Buttigieg2014a; @Buttigieg2014].

\newpage
## Main results and discussion

### Occurrence of QREC
At the start of the project, the only information available on occurrence of QREC in Norwegian animals was in the NORM-VET reports. Here, occurrence data for each year was readily available. However, no overall occurrence comparison between animal species across the years had been performed. The difference in overall occurrence between animal species was therefore largely unknown. Compiling and analysing data from NORM-VET surveillance was therefore regarded as a good starting point, and could give pointers for where further investigation was needed. The results of this investigation showed an overall low occurrence in several animal species in Norway. An increasing trend of QREC occurrence was detected in broilers. The conclusion was that since little to no quinolones are used in these animal species, other factors could play a role in the occurrence of QREC (paper I).

In paper I, different levels of QREC occurrence were detected with the traditional method among commensal _E. coli_ in different production animal species between 2006 and 2016. These animals are fed different feed, stay in different environments, have a different anatomy, and vastly different production site densities. All of these aspects and more may affect the QREC occurrence within these animal populations. No QREC were detected in horses or sheep in the current project. However, QREC were detected in 0.3% of commensal _E. coli_ isolates from sheep in NORM-VET 2018 [@NORM/NORM-VET2018]. An overall QREC occurrence of 0.3% and 0.4% was identified among commensal _E. coli_ from pigs and cattle, respectively. All of these occurrence levels are comparable to the levels reported by other Nordic countries, such as Sweden and Denmark, in the same time frame [@SVARMREPORTS2019; @DANMAPREPORTS2019], see Figure \@ref(fig:nordic). The overall European level of ciprofloxacin resistant _E. coli_ occurrence in pigs and cattle was in 2017 10.6% for both species [@EFSA2017]. This indicates that the Nordic countries have a much lower occurrence level than most other European countries. For broilers, the occurrence seems to be more varied among the Nordic countries. A slightly increasing trend of QREC occurrence has been observed in both Norway and Denmark. Contrastingly, the occurrence in Sweden has been decreasing since 2013 from 14% to 7% in 2018 [@SWEDRES/SVARM2013; @SWEDRES/SVARM2018]. Regardless of these increasing and decreasing trends, the overall occurrence of QREC in the three countries from 2010 to 2018 is relatively similar, at 2.13% in Norway, 2.66% in Denmark, and 2.89% in Sweden based on data from the NORM-VET, SVARM and DANMAP reports [@NORMVETREPORTS2019; @SVARMREPORTS2019; @DANMAPREPORTS2019]. These numbers correspond to a low occurrence in all three countries.

```{r, fig.cap="\\label{fig:nordic}Overall QREC occurrence in broilers, pigs and cattle in Norway, Sweden and Denmark between 2009 and 2018. Data summarized from the NORM-VET, SVARM, and DANMAP reports, respectively \\citep{NORMVETREPORTS2019,SVARMREPORTS2019,DANMAPREPORTS2019}.", echo=FALSE, message=FALSE, fig.pos="H", fig.align="center", out.width="110%", out.height="110%"}
knitr::include_graphics("images/pdfs/qrec_occurrence_nordic.pdf")
```

Norway is, to the author's knowledge, the only country that has done selective screening for QREC among selected animal species.  Comparison of occurrence data generated by this method to other countries is therefore difficult. However, the selective method supplies valuable data when used together with the traditional method. The two methods together provide a rough overview of the relative amount of QREC in the samples tested. For example, using the selective method, an occurrence of 2.4% was detected in samples from horses [@NORM/NORM-VET2017], 7.2% in samples from cattle [@NORM/NORM-VET2015], and 9.6% in samples from sheep [@NORM/NORM-VET2018]. In contrast, the corresponding occurrence in broilers and pigs were 89.5% and 54.3%, respectively [@NORM/NORM-VET2014; @NORM/NORM-VET2015]. Taken together with the occurrences generated with the traditional method presented above, there is an obvious difference in QREC occurrence between cattle, sheep and horses when compared to pigs and broilers. A hypothesis that the anatomy of ruminants might affect the occurrence of QREC in these animal species was investigated in the project. This hypothesis had previously been addressed in a Swedish study. There, they concluded that the prevalence of QREC was higher in calves less than 18 days of age compared to older calves [@Duse2015]. Very young veal calves have not yet developed the intestinal microbiota. They are therefore regarded as monogastric up until the animal stops drinking milk from the mother and starts eating solid food. At this point, the animal is regarded as a ruminant. A small pilot study addressing this hypothesis was performed by collaborators. The temporal occurrence of QREC in calves and their mothers within two different farms were investigated (unpublished data). The calves were sampled frequently during the first three months of life, and the samples were subjected to selective screening of QREC. The results indicated that there were other factors than age alone that affected the occurrence of QREC. However, as this was only a small pilot study, additional studies are needed to further investigate this hypothesis.

In NORM-VET, pigs are sampled individually and the samples are not pooled together as they are for broilers. The occurrences detected with both methods indicate that QREC is present in a high proportion of samples from pigs, but at low levels. The occurrence of QREC in other Nordic countries is similarly very low [@EFSA2017]. A study is currently ongoing at the Institute where pig herds that have been medically remediated with quinolones against _Actinobacillus pleuropneumoniae_ were sampled. These pig herds had been medically remediated only once between three to 25 years ago and were compared to control herds, where quinolones had, with a high certainty, not been used. A semi-quantitative method was used to identify the occurrence and relative presence of QREC in samples from both case and control herds. Preliminary results showed a significant difference between the case and control herds (unpublished data) and indicates persisting QREC in the production environment years after medical remediation with quinolones. However, it is unknown if these QREC isolates were already present before being exposed to the quinolones, or if commensal isolates developed resistance as a response to the exposure. The findings in this ongoing study may indicate that even with low quinolone usage, QREC may be selected for or develop in the gut of these animals and persist in the pig production environment.

In the current study, the occurrence of QREC was decidedly highest in broilers compared to the other production animals included. Samples from broilers are pooled before screening, which may have increased the probability of detecting QREC among the samples tested. Nonetheless, since quinolones are not used in the Norwegian broiler production, a question arose regarding their origin. It was hypothesized that the observed occurrence in broilers was either due to introduction and dissemination of existing strains, or by spontaneous development of quinolone resistance in wild type _E. coli_ in the gut of the animal due to unknown selective pressure. Similarly, because of the observed occurrence of QREC detected with the selective method in pigs, isolates from both species were whole genome sequenced to detect possible dissemination of similar isolates within each production chain. Additionally, QREC isolates from wild animals were included to identify isolates that may be introduced to these production chains from the outside. Moreover, which resistance mechanisms that were dominant among the QREC isolates from these animal populations was not known, thus a resistance mechanism characterization was performed.


### Resistance mechanism characterization and comparative genomics
The results in paper II suggested that the majority of resistance mechanisms identified were chromosomally encoded. The major resistance mechanisms causing quinolone resistance among these isolates were mutations in _gyrA_ and _parC_, which are well described mechanisms in the literature [@Hooper2015]. Novel mutations were identified in some of the other investigated chromosomal genes, but the effect of these mutations is still unknown. _In vitro_ mutational studies are needed to further investigate if these mutations have an effect on MIC values toward quinolones. Transcriptomics may also give useful information on the expression levels of these proteins.

A relatively high occurrence of PMQR was detected among the pig isolates (26.6%) compared to the broiler isolates (4.6%). The low occurrence of PMQR in broilers is in concordance with other Nordic studies on QREC [@Myrenaas2018; @Borjesson2015], and may indicate that PMQR determinants are relatively rare in the Nordic broiler production chain. These concordant results emphasize the dominance of chromosomally encoded quinolone resistance mechanisms in the QREC isolates from the broiler production, and favour the hypothesis that QREC isolates are mainly vertically disseminated in the Nordic broiler production chain. Therefore, PMQR seem to be of minor importance in the development and dissemination of QREC in the Nordic broiler production chain. In contrast, PMQR genes, specifically _qnrB19_  and _qnrS1_, were detected in a high proportion of the pig isolates. _qnrB19_ have previously been identified on small, non-conjugative plasmids harbouring little to no other resistance genes conferring resistance toward other antimicrobials [@Soares2019], while _qnrS1_ has been associated with large, conjugative plasmids harbouring multiple resistance genes [@Dolejska2011a; @Slettemeas2019]. Plasmid characterization has not been done in the current study, but if the _qnrB19_ genes are indeed located on non-conjugative plasmids, then this may indicate that they are clonally disseminated in the pig production chain.

Phylogenetic analysis of QREC isolates from broilers and pigs revealed possible dissemination within the respective production chains. Moreover, possible persistence in the broiler production chain was detected. Additionally, highly similar major sequence types among isolates from broilers were detected. These findings, together with the low occurrence of PMQR in broilers, suggested a clonal dissemination of QREC in the broiler production chain. The data thus suggested that dissemination was a major contributor to the observed occurrence of QREC in the broiler production chain. However, it was unknown if these strains were disseminated from higher levels of the production chain as suggested by a Swedish study [@Borjesson2015], or if the resistance developed in wild type isolates somewhere in the Norwegian broiler production chain due to an unknown selective pressure. To investigate this further, both QREC and wild type _E. coli_ were whole genome sequenced and compared (paper III). 

The results in paper III further supported the hypothesis that QREC is disseminated through the Norwegian broiler production chain, and likely originate from imported breeding animals. This hypothesis has also been investigated in other Nordic countries both for QREC and cephalosporin-resistant _E. coli_, where the authors reach a similar conclusion as the present study [@Myrenaas2018; @Borjesson2015; @Nilsson2014; @Bortolaia2010a; @Petersen2006; @Mo2014]. Major QREC STs, the same as the ones identified in the current study, was reported in other Nordic countries[@Myrenaas2018], namely ST355, ST10, and ST349. These results, taken together with the similar levels of quinolone usage among these countries, provides strong evidence for implicating import of breeding animals as a major causative factor for the occurrence of QREC in the Norwegian broiler production. However, only a few samples from parent flocks were included in this project. To confirm that the major QREC lineages are introduced to the Norwegian broiler production through imported breeding animals, further sampling and characterization of QREC from parent flocks need to be performed. Also, studies comparing isolates from several Nordic and European countries would be of interest. To the authorsâ€™ knowledge, no data on QREC occurrence or genetic characterization of QREC is available from the grandparent distributor in Scotland or Germany. This information would further provide much needed data on this hypothesized international dissemination.

The possibility of quinolone resistance development in wild type _E. coli_ have, to the author's knowledge, not previously been investigated by comparing QREC and wild type isolates using a phylogenetic approach. The result of this analysis in paper III indicated that this rarely occurs in the Norwegian broiler production chain. However, one instance was observed where quinolone resistance was hypothesized to develop in a wild type _E. coli_, where the two isolates were isolated eight years apart, from broilers from different production sites. Because of the geographical distance, it was hypothesized that the wild type isolate may have either been disseminated to a lower level of the broiler production before developing resistance, or developed resistance at a higher level in the broiler production and were subsequently disseminated. However, it was impossible to conclude on this from the current data. Because of the low occurrence of such resistance development among the isolates, it is likely that the hypothesized selective pressure is not very strong, as only a low level of QREC have been detected among the QREC positive samples. Taken together with the evidence for import and dissemination described above, the development of quinolone resistance in wild type _E. coli_ seem to be of minor importance for the occurrence of QREC in the Norwegian broiler production.

Other than the work presented in paper II and in the upublished study, no further investigation of the presence of QREC in the Norwegian pig production was performed in this project. Since the Norwegian pig production is domestic, QREC is less likely to be introduced to the pig production chain in a similar manner as the broiler production, since little to no live pigs are imported each year. Feed has previously been found to introduce QREC strains to the broiler production chain on Iceland [@Thorsteinsdottir2010]. However, no QREC were identified in pig feed in Norway by using the selective method in 2016 [@NORM/NORM-VET2016], which makes it less likely as a source of QREC in the pig production. Further studies are needed to investigate the source of QREC in the Norwegian pig production chain.
