# Discussion


## Methodological considerations

### Selection bias
All isolates included in this study were collected through the NORM-VET programme, isolated with either the non-selective or selective method (see Section \@ref(norwmonitor)). The NORM-VET sampling regime ensures a representative selection of the actual _E. coli_ population in the broiler- and pig production chains, as well as wild animals, due to pre-defined sampling schemes. In the epidemiological study, all isolates of _E. coli_ and QREC from the NORM-VET programme between and including 2006 to 2016 was selected. Therefore, the included samples were deemed representative for the actual _E. coli_ populations in their respective animal species. In the QREC HTS study (article 2), selection was done to ensure phenotypic diversity of the isolates. Isolates were grouped based on phenotypic traits (MIC-values). Therefore, the observed diversity may not represent the actual genomic diversity in the QREC populations within each of the included animal species. In the broiler study, isolates were included if the location of origin was sampled at least three times, and that the location was represented with at least one QREC and one sensitive isolate.


### Genomic analyses methods
The various software used in the genomic analysis was selected because they are known and reliable, and most of them are implemented in the in-house pipeline Bifrost (github.com/NorwegianVeterinaryInstitute/Bifrost).

#### Contaminants
MALDI-TOF was used to confirm the species as _E. coli_ during DNA extraction. However, some samples were found to be contaminated with _Citrobacter_ or _Enterobacter_ using Mash. Inoculating impure samples may have resulted in extraction of DNA from multiple species, as some bacteria may have similar morphology on MacConkey media. This contamination resulted in the exclusion of four isolates due to high amounts of reads originating from these species. Reads from other bacterial species than _E. coli_ may also have been present in other isolates. However, filtering contigs smaller than 500 bp after assembly should remove most of the contaminant contigs, and increased the overall quality of the assemblies.

#### Identification of genes and mutations
In this project, ARIBA [@Hunt2017] was used to identify resistance genes, mutations in intrinsic genes related to resistance, and for sequence typing. For this work, ARIBA has multiple advantages, such as being able to detect both resistance genes and mutations. ARIBA is a read-based program, and using a read-based approach for the detection of AMR genes has been found to be superior to using assembled sequences [@Clausen2016]. The process of genome assembly is complicated, and if one gene is for example split over multiple contigs they may be missed [@Hunt2017;@Clausen2016]. Another advantage of using ARIBA is the possibility of downloading newly updated databases of our choosing. Here, the ResFinder [@Zankari2012] database was used for identification of acquired genes, and the MegaRes [@Lakin2017] database for mutations in chromosomal genes. Resfinder is a popular database for acquired resistance genes, which is manually curated and updated regularly. MEGARes was originally created for metagenomic characterization of AMR genes and mutations [@Lakin2017], and contains references from ResFinder, ARG-ANNOT, the Comprehensive Antibiotic Resistance Database (CARD), in addition to the National Center for Biotechnology Information (NCBI) Lahey clinic $\beta$-lactamase archive. MEGARes is also manually curated, and is specifically annotated for high-throughput data processing [@Lakin2017]. MEGARes was chosen as a database because it already encompasses other popular resistance databases, and is easily downloaded through ARIBA.

When working with databases, one should be aware of the limitations they infer on the resulting data. In the case of acquired resistance, one can only identify the genes which are included in the database. Potential novel acquired genes are therefore missed. However, some mutations or genes may have been misclassified as not present due to the handling of the flags reported by ARIBA. ARIBA provide flags to assess the quality of the identified gene or mutation. In the present study, flags that did not represent the inclusion criteria were excluded from the final results. Thus, some isolates may have been categorized as negative for the given variant or gene, when in reality the gene or variant were not of adequate quality.


### Phylogeny
Phylogenetic methods were used to identify the evolutionary relationship between the isolates. An evolutionary approach was used because the isolates had no clear epidemiological link. In an outbreak situation, it is to be assumed that the isolates involved have a recent common ancestor - this may not be the case in situations where the isolates are not expected to be closely related. The workflow in an outbreak setting can therefore be described as a "top-to-bottom" approach, where clonality of the isolates is already assumed, and typing-based methods like MLST or core genome MLST (cgMLST) may be sufficient. In the case of a non-outbreak setting, clonality is not assumed, and a "bottom-to-top" approach is used. Here, more detailed methods are needed to define isolates as clones, as typing-based methods do not take into account the amount of difference between alleles [@Feil2010]. Therefore, a phylogenetic approach was used to get a deeper resolution when determining the relationship between isolates.

Genomes were assembled with SPAdes prior to phylogenetic analysis. When dividing each read into _k_-mers, the difficulty of putting them back together is increased by the _k_-mers that contain sequencing errors. These _k_-mers will create diverging paths in the de Bruijn graph, further complicating the assembly. The algorithm may then take an erroneous path through the graph, resulting in an inaccurate concensus sequence. A perfect eulerian path is not possible, as sequencing errors and repeats obscure the graph [@Pevzner2001]. These errors may introduce or mask SNPs or indels in the assembly, which may have influenced the phylogenetic analysis. Pilon was used to correct errors by mapping the reads back to the assembly. However, even with this correction step, errors may persist and influence the topology or branch lengths in the final trees.

Removing recombinant sites is an important step when analysing vertical inheritance. Here, Gubbins was used in this regard, which have been found to be much faster than similar software [@Croucher2015], such as ClonalFrameML [@Didelot2015].

IQTree was selected as the ML algorithm to find the optimal phylogenetic tree. Comparative analyses have shown that the search strategy implemented in IQTree achieves higher likelihoods than similar algorithms [@Nguyen2015], such as RAxML [@Stamatakis2014] and PhyML [@Guindon2010], and IQTree seem to find optimal trees faster. Additionally, IQTree is easier to use for non-evolutionary biologists compared to RAxML, which was the main incentive in selecting IQTree. However, IQTree was not always the best algorithm in the comparative analysis between RAxML, PhyML and IQTree, and the authors recommended using all three programs [@Nguyen2015]. In this study, only IQTree was used, which means that potential trees with higher likelihoods may have been missed. 

An evolutionary model was selected by using ModelFinder plus [@Kalyaanamoorthy2017] implemented in IQTree for each phylogenetic analysis. ModelFinder finds the optimal evolutionary model for the data, based on criteria that control for overparameterization [@Kalyaanamoorthy2017]. This way, the data itself choose which model is appropriate, rather than the user. However, using more than one model may be necessary when working with many isolates from potentially different lineages, as some lineages may evolve differently than others. The selected models used in this study may therefore not be optimal in describing the evolutionary rate of all the isolates in one tree. This is also partly why deeper phylogenetic analysis was performed on more closely related isolates separately, such as the ST117 isolates in Article II and ST355 isolates in Article III, as other more appropriate evolutionary models were determined for these trees. 


### Statistical considerations

A non-parametric iteration test was used to assess the aggregation of samples in phylogenetic trees based on SNP-distances. A non-parametric test was selected because the underlying distribution of SNP-distances did not follow a theoretical distribution, such as a normal distribution. Therefore, a test that does not rely on distribution assumptions was needed. 


Non-metric multidimensional scaling (NMDS) was used to cluster isolates based on presence/absence of resistance genes and mutations (Article II). NMDS is an ordination technique that differs from other ordination methods, such as principal component analysis (PCA), as NMDS does not assume a relationship between the samples [@Holland2008]. Therefore, any distance measure is allowed in NMDS, which makes this method well suited for a wide variety of data. NMDS is an iterative method that finds a solution after _n_ number of iterations or after an acceptable solution has been found. This means that running a separate NMDS analysis on the same data may yield a slightly different result. This is done to reduce the computational power needed in the analysis. However, this also means that the "best" solution may not be found, as the analysis may be stuck in local minima and only identifies the best solution of all nearby solutions [@Holland2008]. Therefore, the solution found in Article II may thus represent an adequate solution rather than the "best" one. A stress value is calculated for each iteration and represents how good the data is summarized by the ordination. In the present study, this stress value was low (< 0.05), which is regarded as a good representation [@Buttigieg2014a;@Buttigieg2014]. When running an NMDS analysis, it is important to select the amount of axes used in the calculation, as this may affect the results. In the present study, two axes were used (the default value), as no decrease in stress was observed by including more axes in the analysis. 




## Main results and discussion

### Occurrence of QREC
At the start of the project, the only information available on QREC in Norwegian animals were in the NORM-VET reports. Here, occurrence data for each year was readily available, but no overall occurrence comparison between animal species across the years had been performed. Therefore, little was known about possible temporal trends within animal species, or the difference in overall occurrence between animal species. Compiling and analysing data from NORM-VET surveillance was therefore regarded as a good starting point, as this may give pointers for where further investigation is needed. The results of this investigation showed an overall low occurrence in several animal species in Norway, both wild and domestic. An increasing trend of QREC occurrence was detected in broilers. The paper concluded that since little to no quinolones are used in these animal species, other factors may pay a role in the development of QREC in these animal species.

No QREC were detected in horses, reindeer or sheep. However, QREC was detected in 0.3% of the included sheep samples in NORM-VET 2018 [@NORM/NORM-VET2018]. Using the selective method, an occurrence of 2.4% was detected in samples from horses in 2017 [@NORM/NORM-VET2017]. No selective screening of sheep or reindeer have been performed. This low occurrence, even with selective screening in horses, indicate that QREC is rare in these animal species. Reindeer are boreal herd animals that probably have less contact with human waste or other sources of bacteria, which may explain the low occurrence of QREC. A very low to no occurrence of QREC was identified in ruminant species, such as cattle and sheep. A thought arose during the project that the anatomy of ruminants may affect the occurrence of QREC in these animal species, as a higher occurrence of QREC have been reported in veal calves compared to milk cows [@Kaesbohrer2012; @Duse2015]. Very young veal calves have not yet developed the intestinal microbiota, and are regarded as monogastric up until the animal stops drinking milk from the mother and starts eating solid food. At this point, the animal is regarded as a ruminant. A small pilot study addressing this hypothesis was performed, where the temporal occurrence of QREC in calves and their mothers within two different farms were investigated (unpublished data). The results indicated non-age dependent drivers for quinolone resistance, but additional studies are needed to further investigate this hypothesis. 

Different levels of QREC occurrence were detected between the production animal species. These animals are fed different feed, have different lifestyles, anatomy, and production site densities, which may affect dissemination within these animal populations. Furthermore, the production pressure, e.g. the turnover of produced animals, is decidedly highest for broilers than any of the other production animal species. This may partially explain the observed occurrence in broilers, as increased animal contact may contribute to increased chance for QREC infection. A significantly higher occurrence of QREC in broilers and wild birds compared to the other animal species was detected. In wild birds this may not have been an unexpected finding, since wild birds have a high degree of freedom and may be migratory. This allows them to be exposed to other environments, maybe in southern European countries with a much higher overall occurrence of QREC [@EFSA2019; @EFSA2018; @EFSA2017]. However, in broilers, the occurrence could not be attributed to migration or a high degree of movement. The broiler production houses have a high degree of biosecurity, preventing exposure to the outside. Therefore, it was hypothesized that the observed occurrence in broilers was either due to introduction of existing strains from imported breeding animals, as previously hypothesized by BÃ¶rjesson et al. [@Borjesson2015], or by spontaneous development of quinolone resistance in wild type _E. coli_ in the gut of the animal due to unknown selective pressure. To investigate possible introduction and dissemination, QREC isolates from broilers were whole genome sequenced, along with isolates from pigs, red foxes and wild birds.

### Resistance mechanisms and phylogenetic analysis

In paper II, possible dissemination of QREC was investigated, and resistance mechanisms were characterized in 280 QREC isolates from broiler, pigs, red foxes and wild birds. The results gave indications of dissemination of QREC within the broiler and pig production chains. Additionally, very little PMQR was identified among the broiler isolates, in contrast to wild birds and pigs, where a relatively high occurrence of PMQR was detected.

A very low occurrence of PMQR was identified among broiler isolates. This finding was in concordance with other nordic studies on QREC [@Myrenaas2018; @Borjesson2015], and may indicate that PMQR determinants are relatively rare in the Nordic broiler production chain. These concordant results emphasize the importance of chromosomally encoded quinolone resistance mechanisms in the QREC isolates from the broiler production, as these isolates seem to be mainly vertically disseminated. Therefore, PMQR seem to be of minor importance in the development and dissemination of QREC in the Nordic broiler production chain. Imported breeding animals have been implicated in the introduction of resistant _E. coli_ into the broiler production chain in several Nordic countries, both for QREC and cephalosporin-resistant _E. coli_ [@Myrenaas2018; @Borjesson2015; @Nilsson2014; @Bortolaia2010a; @Petersen2006]. These results, in combination with no quinolone usage in the Norwegian broiler production, provides strong evidence for implicating import of breeding animals as a major causative factor for the occurrence of QREC in the Norwegian broiler production. However, only a few samples from parent flocks were included in this project. To confirm that the major QREC lineages are introduced to the Norwegian broiler production through imported breeding animals, further sampling and characterization of QREC from parent flocks need to be performed. Many studies have focused on the national dissemination of both cephalosporin-resistant and quinolone resistant _E. coli_ in the broiler production chain, where they have provided data that points to implicating imported animals [@Borjesson2015; @Nilsson2014; @Mo2014]. Future studies on these matters should therefore focus on comparing isolates from several countries, and investigating these aspects higher in the broiler production pyramid. To the authors knowledge, no data on QREC occurrence or genetic characterization of QREC is available from the grandparent distributor in Scotland or Germany. This information would further provide strong evidence of this hypothesized international dissemination.

No further investigation of the presence of QREC in the Norwegian pig producton was performed in this project, other than the work presented in paper II. Investigating possible causative factors for the presence of QREC in the pig production is just as important, as the production is domestic. QREC can therefore not be introduced to the pig production chain in a similar manner as the broiler production, since little to no live pigs are imported each year. Interestingly, a relatively high occurrence of PMQR among pig isolates was detected. 

The question of where the QREC isolates originate from in the pig production is not yet answered, and additional studies are needed to investigate these aspects further.


Regarding the other chromosomal genes investigated for mutations, no direct effect of these mutations was possible to deduce from the current genomic data. In-vitro mutational studies are needed to further investigate if these mutations have an effect on MIC values toward quinolones. Furthermore, transciptomics may also give useful information on the expression levels of some of these proteins. 


<!-- Something about paper II -->



After identifying evidence for both dissemination and persistence in the Norwegian broiler production, it was decided to further investigate this phenomenon by doing comparative genomics on both QREC and wild type isolates from broilers. 


<!-- Something about paper III -->




<!-- Something about everything -->
































### Summarise and describe existing data on occurrence of quinolone resistance

Generally, the results in Paper I show a low occurrence of QREC in various animal species in Norway, both wild and domestic. The levels of occurrence in Norwegian livestock is lower than the levels reported from the rest of mainland Europe [@EFSA2019]. For instance, broilers were included in the NORM-VET programme in 2006, 2009, 2011, 2014, 2016 and 2018, and had a mean occurrence of quinolone resistance among commensal _E. coli_ of 5.1%. In comparison, the overall occurrence in poultry in Europe in the same time frame was 49.7%. Similarly, the overall occurrence in pig populations in Norway were 0.3% using data from 2007, 2015, 2016 and 2017, while the rest of Europe was at 15% using data from 2007, 2015 and 2017. . Additonally, import of grandparent animals have previously been implicated in the dissemination of QREC in the broiler production chain in Sweden [@Borjesson2015], which may also be the case in Norway. The pig production can be regarded to have the second highest production density of the included livestock species. Interestingly, a distinctly lower occurrence was observed among the pig isolates. One important difference between the pig and broiler production in Norway is that the pig production is regarded as a completely domestic production, with very little import of live animals. Since very little quinolones are used in the Norwegian pig production, it is not clear what the source of the observed QREC may be. One possible route of introduction may be pig feed. 





Wild animals are currently not included in European surveillance programs. Comparison of occurrences between countries using the same laboratory methods is therefore difficult. However, some studies on quinolone resistance in wild animals have been performed. A study of PMQR in _E. coli_ and _Klebsiella_ spp. in Canada used selective screening with 0.5 mg/L ciprofloxacin to identify QREC from faecal samples from corvids, and identified QREC in 18% of the included samples [@Janecko2018]. This is markedly higher than what was detected in NORM-VET using the selective method (5.6%), although with a ciprofloxacin concentration of 0.06 mg/L [@NORM/NORM-VET2016]. In the current project, wild birds had the second highest occurrence of QREC detected with the traditional method. Wild birds represented many different species of birds, some of which were migratory. Migratory birds migrate to southern countries during winter, and are exposed to a different environment at regular intervals. Also, due to their freedom of movement, they may have increased contact with multiple sources of bacteria, such as garbage or sewage. The observed occurrence in wild birds may therefore be a consequence of this increased freedom of movement compared to other animals. In contrast, no QREC were detected in samples from reindeer. This finding was not unexpected, as reindeer are most likely not exposed to as many different environments as wild birds. Furthermore, 








The results in paper I are based on the traditional method used in the NORM-VET programme to identify resistance in commensal _E. coli_. Use of quinolones in Norwegian livestock is generally low, which is reflected in the overall low occurrence of QREC in multiple animal species compared to other countries in Europe [@EFSA2017]. Nonetheless, the selective method introduced in the surveillance program in 2014 revealed a high proportion of QREC-positive samples. Preliminary data on QREC quantification in these samples suggest that the fraction of QREC isolates compared to the _E. coli_ population in the sample is low (data not shown). However, this needs to be confirmed by further analysis to confirm. This indicates that even though a high proportion of samples are QREC positive, only a few of the _E. coli_ are quinolone resistant, which is important to keep in mind when interpreting the occurrence percentage for the selective method.


### Genetic mechanisms that may explain the observed quinolone resistance

Overall, the identified quinolone resistance mechanisms are in concordance with known major mechanisms that cause quinolone resistance in _E. coli_ [@Hooper2015]. The main mechanism by which _E. coli_ gain quinolone resistance is missense mutations in the QRDR of _gyrA_, especially mutations leading to the amino acid substitution S83L. Furthermore, missense mutations in _parC_ and/or _parE_ have often been implicated in conjunction with missense mutations in _gyrA_ [@Lascols2007; @Slettemeas2019; @KompLindgren2003], but rarely on their own. In article II, only two isolates had amino acid substitutions in the QRDR of _ParC_ alone. However, these two isolates harbored the same _qnr_-gene. The presence of _qnr_ genes had a strong negative correlation with the presence of amino acid substitutions in the QRDR of GyrA _r_(278) = -0.91, _p_ < 0.05. This suggests either that _qnr_ genes are not often acquired in isolates already harboring missense mutations in _gyrA_, or that mutations in _gyrA_ do not arise so easily in isolates that already harbor _qnr_-genes. A previous study suggested that if _qnr_-genes were present in isolates with missense mutations in the QRDR of _gyrA_, then most likely the mutation preceeded the _qnr_ gene [@Cesaro2008]. This is likely, because the Qnr proteins protect the topoisomerases from quinolone action, and reduces the amount of lethal quinolone-DNA-topoisomerase cleavage complexes in the cell [@Robicsek2006a]. These findings therefore suggest that the presence of _qnr_-genes prevents mutations from arising in _gyrA_.

PMQR was detected in 21% of the QREC isolates in article II and none of the isolates in article III. This occurrence is in concordance with other studies in Europe [@Veldman2011;@Borjesson2015;@deJong2018;@Wasyl2018]. PMQR could be an explanation for the presence of quinolone resistance without selective pressure, as these genes may be disseminated by horizontal transfer regardless of selective pressure. Other resistance genes conferring resistance to additional antimicrobials may be present on the same plasmid as the PMQR determinant, which may enable co-selection of QREC isolates. _qnr_-genes have regularly been found on plasmids bearing additional resistance genes, such as extended spectrum $\beta$-lactamases (ESBL) [@Strahilevitz2009; @Jacoby2006a; @Zurfluh2015d]. In Article II, a significant correlation between _qnr_-genes and genes conferring resistance to other antimicrobials, such as chloramphenicol and tetracycline, was identified. This positive correlation indicates that these specific resistance genes are present on the same plasmids as _qnr_-genes, and these QREC isolates may therefore be selected for through the use of these antimicrobials as well. However, the use of antimicrobials in the Norwegian broiler production is generally low, regardless of antimicrobial class. Co-selection is therefore not likely, as selective pressure for other antimicrobials are also regarded as negligible.
PMQR usually confer low quinolone resistance on its own compared to chromosomal mutations. Therefore, PMQR could be a plausible explanation of the presence of QREC in Norwegian livestock, as most of the MIC values of the isolates were within 0.12 to 1 mg/L ciprofloxacin. However, the isolates in the present study had a relatively low occurrence of PMQR genes, as well as ESBL and other acquired resistance genes. This occurrence indicate that quinolone resistance gene-bearing plasmids are rare in these animal populations in Norway, and only play a small role in the occurence of quinolone resistance in these animal environments.



The results indicate that the major mechanisms that explain the observed quinolone resistance are mutations in the QRDR of GyrA and ParC, especially the S83L substitution in GyrA. The overrepresentation of chromosomal resistance mechanisms over PMQR indicate that the majority of the quinolone resistance mechanisms are clonally disseminated. This clonal dissemination hypothesis was further supported with the NMDS analysis, which identified the resistance mechanisms to be relatively homogenous within major STs.


### Possible associations between isolates from various origins

The presence of QREC in various production animal species with very low quinolone usage was unexpected and of concern. We hypothesized that by investigating the relationship between the QREC isolates would enable us to gain insights into potential dissemination and/or persistence, and thus be able to identify major contributors to the presence of quinolone resistance in their respective environments. In paper II, a high ST diversity was observed among the included animal species. Despite this diversity, we identified phylogenetically related QREC isolates from pigs sampled from different production sites. Similarly, we also identifed phylogenetically related isolates from broilers, and between broilers and a red fox. Major QREC lineages in the broiler and pig production chain were identified, namely ST349 and ST355, and ST117, respectively.

Dissemination of QREC in the broiler production chain have previously been investigated in other nordic countries [@Bortolaia2010a; @Myrenaas2018; @Petersen2006], and the presence of similar QREC throughout the Swedish broiler production was attributed to imported breeding animals [@Borjesson2015]. In our study we found the same STs as reported in other countries with the same distributor of breeding animals as Norway. The isolates represented in these STs were highly similar, some of which were thought to have a recent common ancestor. Most of these isolates were from different production sites, indicating dissemination from breeding flocks.










In Article II and III, the association between the included isolates were investigated by phylogenetic analysis. The aim this analysis was to identify possible dissemination of QREC strains, and to detect possible development of quinolone resistance in wild type _E. coli_ in the broiler production chain. In short, the results suggests dissemination of major STs through the broiler production chain, and little development of quinolone resistance among wild type _E. coli_. Furthermore, phylogenetically related isolates were detected among broilers and red foxes, from geograhically distant locations and from different years.

The major STs identified among QREC isolates from broilers were ST10, ST349 and ST355. These STs have previously been identified as major STs of QREC isolates in samples from broilers in Norway, Sweden and Iceland [@Myrenaas2018]. The broiler production environment is a highly stable environment with little temperature changes, closed to the outside, and is populated by a single animal species. It is therefore plausible to think that isolates within this environment have little reason to change, due to its stable nature. These major STs may therefore represent isolates that are highly adapted to the broiler production environment, and is likely to persist if introduced through imported animals. Parent animals are imported to Norway from Sweden, which is a country much like Norway in regards to quinolone usage. The overall sale of quinolones for animal use in Sweden have dropped 82% since 2009, and only 29 kg of active substance was sold in 2018 [@SWEDRES2019]. Additionally, a Swedish study have implicated imported breeding birds as the plausible source of QREC in their broiler production chain [@Borjesson2015]. This indicates that QREC isolates may originate from the top of the pyramid. Grandparent animals are imported from Scotland to Sweden, but no data on QREC occurrence or quinolone usage is available from this section of the production pyramid. Furthermore, phylogenetically related QREC from broilers and red foxes isolated in different years and geographic locations were identified, which may indicate persistence of some QREC strains in the broiler production. 

Phylogenetically related isolates from pigs were identified in Article II. These were isolates from geographically distant location, but within the same year. Since import of live animals to the pig production is negligible, the origin of these QREC isolates is likely within the Norwegian pig production. The presence of these isolates may indicate dissemination within the pig production chain as well, likely through either a shared piglet distributor or through satelite units of the sow pool system. 


Feed may be source of QREC [@Thorsteinsdottir2010]. A high diversity may indicate introduction of multiple strains into the production chain [@Bortolaia2010a].

